jenkins/ci.suse.de/openstack-ardana.yaml

```diff
@@ -38,6 +38,12 @@
           description: >-
             The Input Model to use
 
+      - string:
+          name: image_id_compute
```

Comment:
Do we want to have this in the jenkins job? What about scenarios where we mix different compute nodes?
I would prefer to keep that in the heat stack logic and just create different jobs for the scenarios.

Reply:
This PR is a continuation of this one:

https://github.com/SUSE-Cloud/automation/commit/2a703f14b2e15752f2afef05913c26b9f398d30d

Certainly mixing different types of compute nodes makes sense, but the task for now is provide a way to do a basic test with centos. This PR allows me to do that manually, and the one already merged enabled the job for automatic tests:

https://ci.suse.de/job/cloud-ardana8-job-std-min-centos-x86_64/